### ISSUE TITLE
CLI: add a flag for showing each result's alias set

### ISSUE THREAD

**Issue by woodruffw** [pypa/pip-audit#235]
Follow-on from #232.

One potential format:

```
$ pip-audit --show-dupes
Found 2 known vulnerabilities in 1 package
Name  Version ID                              Fix Versions
----  ------- ------------------------------- ------------
Flask 0.5     PYSEC-2019-179, FOOBAR-2019-666 1.0
Flask 0.5     PYSEC-2018-66                   0.12.3
```

We could also do this as a separate `Aliases` column instead, in which case we might want to make the flag `--show-aliases`.

**Comment by di** [comment]
FWIW the actual flag name I used was just a placeholder, I don't think it's a great name for UX.

**Comment by woodruffw** [comment]
CC @jleightcap: this is a good potential first issue, to get familiar with the codebase.

**Comment by pboulos** [comment]
Hello everyone, I've written an implementation for this as I had need for it in a project I was working on (see #721).

I've looked over your contributing docs, but please let me know if you'd like me to go about this another way (and obviously for any adjustments you'd like to make).

**Comment by woodruffw** [comment]
Thanks @pboulos! I'll give this a review tomorrow. 

CCing @tetsuo-cpp as well 🙂

**Comment by woodruffw** [comment]
Merged with #721! Thanks a ton @pboulos!
